web3/contract.py

```diff
@@ -695,7 +708,7 @@ def check_forbidden_keys_in_transaction(
 
 
 class ConciseMethod:
-    ALLOWED_MODIFIERS = {'call', 'estimateGas', 'transact', 'buildTransaction'}
+    ALLOWED_MODIFIERS = {'call', 'estimate_gas', 'transact', 'build_transaction'}
```

Comment:
Same question here about keeping `estimateGas` and `buildTransaction` here.

Reply:
These turn out to be args passed in to the `ConciseMethod.__call__` method so they should have stayed the same since it would be a breaking change. I will change these back.